pil_analyzer/src/pil_analyzer.rs

```diff
@@ -561,11 +593,21 @@ impl<T: FieldElement> PILContext<T> {
                     .get(name)
                     .unwrap_or_else(|| panic!("Constant {name} not found.")),
             ),
-            PolynomialReference(_) => None,
+            PolynomialReference(name) => {
+                // TODO this whole mechanism should be replaced by a generic "reference"
+                // type plus operators.
+                if !name.shift() && name.namespace().is_none() {
+                    // See if it might be a constant
+                    self.constants.get(&name.name().to_owned()).cloned()
+                } else {
+                    None
+                }
+            }
```

Comment:
I don't understand how this is related to this change

Reply:
This is needed for
```
let N = 16;

namespace Fibonacci(N);
```

Currently, `%something` is parsed as a "ConstantReference" while some identifier that does not start with `%` is parsed as a "PolynomialRefernce". In the medium term, I would like to parse both just as a "Reference".